coordinator/src/main/java/org/apache/uniffle/coordinator/ApplicationManager.java

```diff
@@ -75,10 +77,13 @@ public ApplicationManager(CoordinatorConf conf) {
       throw new UnsupportedOperationException("Unsupported selected storage strategy.");
     }
     expired = conf.getLong(CoordinatorConf.COORDINATOR_APP_EXPIRED);
-    QuotaManager quotaManager = new QuotaManager(conf);
-    this.currentUserAndApp = quotaManager.getCurrentUserAndApp();
-    this.appIdToUser = quotaManager.getAppIdToUser();
-    this.defaultUserApps = quotaManager.getDefaultUserApps();
+    if (conf.get(CoordinatorConf.COORDINATOR_ACCESS_CHECKERS).stream().map(String::trim).collect(Collectors.toList())
+        .contains(AccessQuotaChecker.class.getCanonicalName())) {
+      this.quotaManager = new QuotaManager(conf);
+      this.currentUserAndApp = quotaManager.getCurrentUserAndApp();
+      this.appIdToUser = quotaManager.getAppIdToUser();
+      this.defaultUserApps = quotaManager.getDefaultUserApps();
```

Comment:
Can we remove `defaultUserApps` in this class?

Reply:
Yes, this's not used in this class.